OpenRA.Game/Graphics/Viewport.cs

```diff
@@ -55,6 +55,12 @@ public class Viewport
 		bool allCellsDirty = true;
 
 		float zoom = 1f;
+
+		public float[] AvailableZoomSteps
+		{
+			get { return new[] { 2f, 1f, 0.5f, 0.25f }; }
```

Comment:
This should be a readonly field instead of a property.  Right now it creates a new object each time you call the property, which makes more work for the garbage collector.